podvm/files/etc/systemd/system/api-server-rest.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
We also need this file to be symlinked into the `multi-user.target.wants` directory.